.github/workflows/push.yml

```diff
@@ -268,39 +270,44 @@ jobs:
   release-msi-windows:
     needs:
       - release
-    # only one parallel job allowed - used shared S3
```

Comment:
To mi přijde že není potřeba serializovat a může běžet současně s `update-repositories`.(?)

Reply:
Netreba. Len `update-repositories-windows` sa musi spustit az po tomto tasku, co je splnene.